models/lfs_lock.go

```diff
@@ -110,9 +112,24 @@ func GetLFSLockByID(id int64) (*LFSLock, error) {
 }
 
 // GetLFSLockByRepoID returns a list of locks of repository.
-func GetLFSLockByRepoID(repoID int64) (locks []*LFSLock, err error) {
-	err = x.Where("repo_id = ?", repoID).Find(&locks)
-	return
+func GetLFSLockByRepoID(repoID int64, page, pageSize int) ([]*LFSLock, error) {
+	sess := x.NewSession()
+	defer sess.Close()
+
+	if page >= 0 && pageSize > 0 {
+		start := 0
+		if page > 0 {
+			start = (page - 1) * pageSize
```

Comment:
so page=0 and page=1 shall give same data?

Reply:
Yup that's standard in Gitea

Reply:
I didn't recognize to code becuase the usual syntax is `if page == 0 { page = 1 }`, but nevermind.